packages/launchpad/src/types.ts

```diff
@@ -58,4 +59,5 @@ export interface LaunchpadTaskActionsProps {
 	extraActions?: {
 		setShareSiteModalIsOpen?: ( isOpen: boolean ) => void;
 	};
+	uiContext: 'calypso';
```

Comment:
I think this should be optional. It's breaking some tests.
```suggestion
	uiContext?: 'calypso';
```
<img width="1003" alt="Screen Shot 2023-08-01 at 09 43 44" src="https://github.com/Automattic/wp-calypso/assets/1234758/b0368d77-1078-4334-95d9-6a003f7cb9df">

Reply:
I addressed this in [b283655](https://github.com/Automattic/wp-calypso/pull/80090/commits/b283655032172224d62cf5780bfdbbe849155901).